src/components/MnemonicDisplay/MnemonicDisplay.vue

```diff
@@ -1,17 +1,20 @@
 <template>
-  <div class="mnemonic_container">
-    <div class="mnemonicWordDiv clear">
-      <span v-for="(word, index) in words" :key="index" class="ghost">
-        <Tag closable @on-close="removeWord(word)">
-          {{ word }}
-        </Tag>
-      </span>
-    </div>
+  <div>
+    <slot>
+      <div class="mnemonic-container">
+        <div class="mnemonic-list">
+          <span v-for="(word, index) in words" :key="index">{{ word }}</span>
+        </div>
+        <CopyToClipboard :prepare-copy-data="words" class="copy-button" />
```

Comment:
- rename `prepare-copy-data` to `v-model` (only possible if `CopyToClipboard` uses "value" prop.)
- instead of passing the `words` _array_, add a getter method `public get wordsText(): string` which takes care of _joining the array with spaces in between_.